doc/source/user_guide/io.rst

```diff
@@ -4658,26 +4658,41 @@ See the `Full Documentation <https://github.com/wesm/feather>`__.
 
 Write to a feather file.
 
-.. ipython:: python
-   :okwarning:
+.. TODO(Arrow 0.15): remove change these back to .. ipython blocks.
 
-   df.to_feather('example.feather')
+.. code-block:: python
 
-Read from a feather file.
+   >>> df.to_feather('example.feather')
 
-.. ipython:: python
-   :okwarning:
+Read from a feather file.
 
-   result = pd.read_feather('example.feather')
-   result
+.. code-block:: python
```

Comment:
Just out of curiosity any reason this isn't an ipython block? Seems like it could reduce code required

Reply:
Because released feather (pyarrow) fails with this PR due to the removed SparseDataFrame (in pyarrow master and upcoming 0.15 release this is already fixed). 

That reminds: Tom, we discussed adding a dummy SparseDataFrame as we did for Panel to avoid direct breakage like this?

Reply:
I don't recall that's a good idea. Then we're not losing a bunch of our feather test coverage while waiting for the next pyarrow release.